### DISCUSSION TITLE
Sidebar options

### DISCUSSION THREAD

**Discussion by jackvermeulen** [vikramlearning/blazorbootstrap#284] [Q&A] [answered]
Hello, 
I use the sidebar element, and I want to add custom icons and if possible add a class per navigation item. Is this possible?

I can't add custom icons with CustomIconName, who has an idea?

**Comment by gvreddy04** [comment, marked as answer]
@jackvermeulen This is addressed as part of **SidebarItem without childs, CustomIconName not passed to Icon #286**
